tests/integration/test_interaction.py

```diff
@@ -305,7 +354,7 @@ def _get_scenario_xml(self):  # pylint: disable=no-self-use
         return "<vertical_demo><drag-and-drop-v2/></vertical_demo>"
 
 
-class BasicInteractionTest(DefaultDataTestMixin, InteractionTestBase):
+class BasicInteractionTest(DefaultDataTestMixin, InteractionTestBase, BaseIntegrationTest):
```

Comment:
@mtyaka Why does this need to extend `BaseIntegrationTest` (genuine question; I might be missing something)?